attributes/default.rb

```diff
@@ -19,3 +19,12 @@
 #
 
 default['ms_dotnet']['timeout'] = 600
+
+if platform? 'windows'
+  nt_version = node['platform_version'].to_f
+  if nt_version >= 6.2
+    default['ms_dotnet']['feature']['enable_all']            = true
```

Comment:
Hi @Annih, this is not strict at all. :+1:  I am all for good and clean structure for easy maintenance. My initial thought was to put it into the ms_dotnet3 attribute file as well since it was already structure that way. It is good to collaborate and share ideas. I will make the change local and test again.